pom.xml

```diff
@@ -16,8 +16,8 @@
     <properties>
         <java.version>17</java.version>
         <mapstruct.version>1.5.3.Final</mapstruct.version>
-        <openapi.version>1.6.14</openapi.version>
         <testcontainers.version>1.17.4</testcontainers.version>
+        <open-api.version>1.7.0</open-api.version>
```

Comment:
The issue with that HTTP was that you were using the version 2.x from openApi, and this version is for spring-boot 3.x not 2.7.x: https://springdoc.org/v1/